users/templates/users/auth_test.html

```diff
@@ -0,0 +1,20 @@
+{% extends "base.html" %}
+
+{% block title %}PolarBear -- sign up{% endblock %}
+
+{% block content %}
+<p>Login/Signup Successfully!! You will be redirected to home page in <span id="sp">4</span> seconds...</p>
+<script>
+
```

Comment:
this page wasn't meant to be permanent. when we have a homepage, we should replace it. this file should be deleted.

Reply:
fixed